src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -8830,6 +8836,30 @@ else if (responseText.contains("You throw the handful") || itemSuccess) {
         }
         return false;
 
+      case ItemPool.SPOOKY_VHS_TAPE:
+        if (responseText.contains("grow wide as they view the contents") || itemSuccess) {
```

Comment:
Not a request for a change, but an observation on all the similar usages in this method:
itemSuccess is a boolean that says whether haiku or anapest messages appeared.
Which is to say, at this point, it's rare - but isn't a haiku katana a LoL item?

Even so, checking itemSuccess first would short circuit all the string.contains checks in this method.
In other words, but the expensive checks after the trivial boolean check, rather than before it.

Reply:
That's a good point and a good idea